support/models/src/local/namespace.ts

```diff
@@ -5,9 +5,99 @@
  */
 import { LocalMatter } from "../local.js";
 
-// Namespace is a standard or device specific Enum ID, so lets handle as normal uint8 number for now
+// Namespace is a standard or device-specific Enum ID
 LocalMatter.children.push({
     tag: "datatype",
     name: "namespace",
-    type: "uint8",
+    type: "enum8",
+    children: [
```

Comment:
Can we instead generate this using Matter.semanticNamespaces.map()?  Then we won't have to worry about copying in new namespaces in the future